cloudify_cli/commands/executions.py

```diff
@@ -100,7 +100,7 @@ def start(workflow_id, deployment_id, timeout, force,
 
     events_logger = get_events_logger()
 
-    events_message = "* Run 'cfy events list --include-logs " \
+    events_message = "* Run 'cfy events list --tail --include-logs " \
```

Comment:
While there is nothing wrong with adding --tail to all events list commands, it is only relevant for timed out executions. If the execution finished (successfully or failed) adding --tail makes no difference and may be somewhat confusing